go/cmd/gh-ost/main.go

```diff
@@ -149,6 +150,13 @@ func main() {
 	if migrationContext.SwitchToRowBinlogFormat && migrationContext.AssumeRBR {
 		log.Fatalf("--switch-to-rbr and --assume-rbr are mutually exclusive")
 	}
+	if migrationContext.TestOnReplicaSkipReplicaStop {
+		if !migrationContext.TestOnReplica {
+			log.Fatalf("--test-on-replica-skip-replica-stop requires --test-on-replica to be enabled")
```

Comment:
As far as I can tell, the above is happening before the migration.  This is in already in `main.go`, and the migration starts on [line 198](https://github.com/Wattpad/gh-ost/blob/2e43718ef38032b9589c32e59c2e9174d9fe44f7/go/cmd/gh-ost/main.go#L198).  Unless I am missing something...